meinberlin/apps/documents/templates/meinberlin_documents/document_dashboard.html

```diff
@@ -8,7 +8,12 @@ <h1 class="u-first-heading">{% translate 'Edit document' %}</h1>
 
 {% endblock %}
 
+{% block extra_css %}
+    {{ block.super }}
+    {{ ckeditor_media.css }}
+{% endblock extra_css %}
 {% block extra_js %}
   {{ block.super }}
+    {{ ckeditor_media.js }}
```

Comment:
why not use script tag or add comment to explain?

Reply:
it already comes as script tag

Reply:
I added a commit to make it clearer

Reply:
I guess mostly because if the `django-ckeditor-5` package changes the name or the path of the js files it will still work. If we hardcode the path it will break